pkg/controllers/dlb/controller_test.go

```diff
@@ -149,3 +153,26 @@ func TestNewDaemonSetDLB(t *testing.T) {
 		t.Errorf("expected and actuall daemonsets differ: %+s", diff.ObjectGoPrintDiff(expected, actual))
 	}
 }
+
+func TestNewDaemonSetDLBWithTolerations(t *testing.T) {
+	plugin := &devicepluginv1.DlbDevicePlugin{}
+	plugin.Name = "testing"
+
+	plugin.Spec.Tolerations = []v1.Toleration{
+		{
+			Key:      "foo",
+			Operator: "Equals",
+			Value:    "bar",
+			Effect:   "NoSchedule",
```

Comment:
Test all keys (in all plugins)?

Reply:
All keys within the struct?

Reply:
It's already almost all of them, so I thought why complete the set... Not really important though.